test/commands/getset.spec.js

```diff
@@ -59,8 +60,7 @@ describe("The 'getset' method", function () {
                 describe("when the key exists in Redis", function () {
                     beforeEach(function (done) {
                         client.set(key, value, function (err, res) {
-                            helper.isNotError()(err, res);
-                            done();
+                            helper.isNotError(done)(err, res);
```

Comment:
I don't know why these take a callback.... they are synchronous. I think we should remove that ability in the future. If someone adds an assertion after this line, they have to know to move the done down. However, if we leave the done where it was and pass no callback, it will be obvious to place a new assertion above the done. IMO, these should all be changed back so that no callbacks are passed to assertion helpers.